### PR TITLE
Big Data Resources

### PR DESCRIPTION
**Pull request by vimoxshah**
Big Data Tutorial
[https://github.com/vimoxshah/Learning-Resources/tree/master/BigData]

### CODE REVIEW

File: BigData/BigDataTutorial.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ [Big Data Projects](https://www.acquia.com/examples-big-data-projects)
+ [Mapr Hadoop White Papers](https://www.mapr.com/resources/white-papers)
+ [Udacity Courses](https://www.udacity.com/courses/data-science)
+ [Coursera Big Data Specialization](https://www.coursera.org/specializations/big-data)
+ [Cloudera Hadoop Certification](http://www.cloudera.com/training/certification.html)
+ [Mapr Hadoop Certification](https://www.mapr.com/services/mapr-academy/certified-hadoop-developer)
+ [Hortonworks Certification](http://hortonworks.com/training/certification/)
+ <br/>
+
+ These lists is small but more will be added soon.
```
Comment (PSanni) [inline BigData/BigDataTutorial.md]: @vimoxshah @PareshMayani This is not necessary to include i think. :smile:

**Comment by PSanni** [comment]
@vimoxshah Try to add description to avoid confusions. Like it's creating confusion in Top 3rd entry(blog).

File: BigData/BigDataTutorial.md (hunk 1 above)
Comment (PareshMayani) [inline BigData/BigDataTutorial.md]: @PSanni yes I agree! Please don't write such sentences. At this stage this repo is having small lists in each directories but all the lists are going to be big soon ;)

**Comment by PareshMayani** [comment]
Hi @vimoxshah, thank you for sharing a list of tutorials/articles. This contribution can be shared in a single commit than three commits.

Second, directory architecture should be like given in "android", check https://github.com/GDGAhmedabad/Learning-Resources/tree/master/android

Please send a new PR with a single commit following formatting guidelines and with proper details for each links.

**Pull request closed without merging**